### ISSUE TITLE
Wave file can expose property of bits-per-second & channels

### ISSUE THREAD

**Issue by ladyada** [adafruit/circuitpython#1303]
right now you can get sample_rate as a property, but not bits/channels (which could help auto-config the mixer/audio playback

https://circuitpython.readthedocs.io/en/latest/shared-bindings/audioio/WaveFile.html

**Comment by tannewt** [comment]
Yup! Good idea! This is a good first issue because one can follow how `sample_rate` is done.

**Comment by ladyada** [comment]
@caternuson heya wanna take a look at adding this? lil different than userland but a good first issue if you are interested in poking into internals!

**Comment by caternuson** [comment]
@ladyada just seeing this mention. sure. let me take a look. i'll respond back here if i think i'll do more damage than good. :)

**Comment by ladyada** [comment]
sweet!

**Comment by ladyada** [comment]
read-only for both!
i think `wave.channels` and `wave.bits_per_sample` are good, @tannewt any thoughts

**Comment by caternuson** [comment]
@tannewt Is read-only as simple as don't have a setter func and set 2nd `.proxy` entry to `mp_const_none_obj`? Like this?
```cpp
const mp_obj_property_t audioio_wavefile_bits_per_sample_obj = {
    .base.type = &mp_type_property,
    .proxy = {(mp_obj_t)&audioio_wavefile_get_bits_per_sample_obj,
              (mp_obj_t)&mp_const_none_obj,
              (mp_obj_t)&mp_const_none_obj},
};

```

**Comment by tannewt** [comment]
@caternuson Yup! That and documenting it as such.

**Comment by caternuson** [comment]
OK - we can continue chatting over in the PR: #1311

**Comment by caternuson** [comment]
Fixed by #1311
